### PR TITLE
[W35] Ray Lin: 08/29 ~ 09/02

### PR DESCRIPTION
**Pull request by raylin**

### CODE REVIEW

File: RayLin.md [hunk 1]
```diff
@@ -1,33 +1,15 @@
...
-      - closed caption button
-      - fullscreen unavailable
-      - all Fennec functionality
-    - TODO
-      - refine patch
-      - final check with UI/UX
+    - r?jaws
+    - Locale: position/duration time string not localizer friendly.
+      - revised and retain original behavior.
+  - Try/Study video testing
```
Comment (timdream) [inline RayLin.md]: Do you need any specific help on this?

File: RayLin.md [hunk 2]
```diff
@@ -1,33 +1,15 @@
...
-below is my outline draft:
-
-  - Write and Test XUL/XBL without rebuilding firefox
-  - Basic XBL concept
-  - Namespce
-  - Inspect XBL binding/anonymous content in content
-  - Inheritance
-  - De-XUL
-
+  Prepare slide in W36
```
Comment (timdream) [inline RayLin.md]: 👍

File: RayLin.md (hunk 1 above)
Comment (raylin) [inline RayLin.md]: Still not go deep into it, but I can foresee video has its complication to increase test coverage, such as different states, sources, format and sizes, etc. I'll see what kind of test video control lacks of and find out what resources is needed by then.

Comment (timdream) [inline RayLin.md]: Could you start filing bugs on the tests you plan to implement?

Comment (raylin) [inline RayLin.md]: No problem :)

**Pull request merged**